portfolio/src/main/webapp/index.html

```diff
@@ -42,9 +42,17 @@ <h1 style=font-size:40px class="heading-font">Apoorva Dhanala's Portfolio</h1>
       <h3 class="links-font"><a href="resume.html">MY RESUME</a></h3>
       <h3 class="links-font"><a href="photography.html">MY PHOTOGRAPHY</a></h3>
       <h3 class="links-font"><a href="favorites.html">FAVORITE SONGS, TV SHOWS, AND MORE!</a></h3>
-      <h3>Click here to see a random quote:</h3>
-      <button onclick="updateGreeting()">Hi!</button>
-      <div id="quote-container"></div>
+    </div>
+    <div class="comments-box">
+      <h2 style=font-size:40px class="heading-font">COMMENTS</h2>
+      <button onclick="updateGreeting()">Click to see all the comments!</button>
+      <div class="links-font" id="quote-container"></div>
+      <form action="/data" method="POST">
+        <h3 class="links-font">Please leave me a comment!</h3>
```

Comment:
fonts are inherited so I think you don't need to respecify links-font (which may need to be renamed given that it doesn't just apply to links any more?) given that it's already specified in an ancestor  element (line 49)

Reply:
I tried that (without respecifying) but for some reason it's not automatically applying it!

Reply:
hmm! thanks for checking!